src/paho/mqtt/client.py

```diff
@@ -3740,7 +3968,18 @@ def _get_proxy(self):
         # None to indicate that the connection should be handled normally
         return None
 
-    def _create_socket_connection(self):
+    def _create_socket(self) -> SocketLike:
+        tcp_sock = self._create_socket_connection()
+        sock = self._create_ssl_socket_if_enable(tcp_sock)
+
+        if self._transport == "websockets":
+            sock.settimeout(self._keepalive)
+            return WebsocketWrapper(sock, self._host, self._port, self._ssl,
+                                    self._websocket_path, self._websocket_extra_headers)
```

Comment:
Would maybe read easier as 
```suggestion
            return WebsocketWrapper(
                sock,
                self._host,
                self._port,
                self._ssl,
                self._websocket_path,
                self._websocket_extra_headers,
           )
```
(and I'd also consider adding kwargs, so it's harder to accidentally pass things in the wrong order)